Create SuperSimpleBTCPaperWallet.html

EAWF opened the pull request:
Something I threw together.

QRCode Display thanks to: https://github.com/davidshimjs/qrcodejs

EAWF committed:
Create SuperSimpleBTCPaperWallet.html

QRCode Display thanks to: https://github.com/davidshimjs/qrcodejs

EAWF committed:
Update SuperSimpleBTCPaperWallet.html

"bitcoin:" prefix not needed for QRCode of Private Key.

coinables commented:
I like this. Good to provide examples for others. Can you create a new folder called examples, then place this HTML inside the examples folder.

EAWF commented:
Ok, but I just need to figure out how to do that.

EAWF committed:
Create SuperSimpleBTCPaperWallet.html

moving file

EAWF committed:
Update SuperSimpleBTCPaperWallet.html

Sample Paper Wallet w/o checking

EAWF committed:
Delete SuperSimpleBTCPaperWallet.html

Moved to examples subdirectory

EAWF committed:
Create btcwallet.html

Wallet Creator for MIT App Inventor or AppyBuilder WebViewer Component tool for creating a BTC Wallet and Checking for existing values automatically, returning the wallet and values to the App.

EAWF commented:
Did I do it correctly?

On 2018-11-05 19:42, Kyle Honeycutt wrote:
> I like this. Good to provide examples for others. Can you create a new
> folder called examples, then place this HTML inside the examples
> folder.
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub [1], or mute the
> thread [2].
> 
> Links:
> ------
> [1] https://github.com/coinables/buidljs/pull/5#issuecomment-436112158
> [2]
> https://github.com/notifications/unsubscribe-auth/AS7G4Zlbk_QROs1l9ErF3DELGzQNXaihks5usPcpgaJpZM4YDMnS

-- 
Bob Holden
EAWF Support Desk
253.576.1686

NOTICE: This email and any files transmitted with it are confidential 
and intended solely for the use of the individual or entity to whom they 
are addressed. This message may contain confidential information and is 
intended only for the individual named. If you are not the named 
addressee you should not disseminate, distribute or copy this e-mail. 
Please notify the sender immediately by e-mail if you have received this 
e-mail by mistake and delete this e-mail from your system. If you are 
not the intended recipient you are notified that disclosing, copying, 
distributing or taking any action in reliance on the contents of this 
information is strictly prohibited.

WARNING: Although the company has taken reasonable precautions to ensure 
no viruses are present in this email, the company cannot accept 
responsibility for any loss or damage arising from the use of this email 
or attachments. The recipient should check this email and any 
attachments for the presence of viruses. E-mail transmission cannot be 
guaranteed to be secure or error-free as information could be 
intercepted, corrupted, lost, destroyed, arrive late or incomplete.
